app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -44,7 +42,29 @@ angular.module('Bastion.components').factory('Nutupane',
                 return $location.path().split('/').join('-').slice(1);
             }
 
+            function setQueryStrings() {
+                if (params.paged) {
+                    $location.search("page", params.page).replace();
+                    $location.search("per_page", params['per_page']).replace();
+                }
+
+                if (params.search) {
+                    $location.search(self.searchKey, params.search).replace();
+                }
+
+                if (params['sort_by']) {
```

Comment:
Why use `[]` for access with this parameter when you used the dot operator for `params.search` ?